accel_code/utils.py

```diff
@@ -162,6 +162,9 @@ def make_directory(old_path, new_path, replace):
             raise ValueError('replace option was not specified and output file exists', new_path)
         if replace == 'yes':
             os.remove(new_path)
+            # Create relevant directories for new file
+            os.makedirs(os.path.dirname(new_path))
```

Comment:
If the file path exists; we should not need to make the path to the file again, since `os.remove` only removes the file at the end of the path, but no directories leading up to the path.